apollo-router/src/plugins/progressive_override/snapshots/apollo_router__plugins__progressive_override__tests__non_overridden_field_yields_expected_query_plan.snap

```diff
@@ -12,14 +12,14 @@ expression: query_plan
           "kind": "Fetch",
           "serviceName": "Subgraph2",
           "variableUsages": [],
-          "operation": "{percent0{foo}}",
+          "operation": "{ percent0 { foo } }",
```

Comment:
Whitespace changes are expected, and we’ve decided not to try and eliminate or reduce them.